docs/annotate.rst

```diff
@@ -53,7 +57,7 @@ Step-by-step
 
    - Click ``>`` to move it to the right column, then click ``OK``.
 
-#. Let’s now add Key-Value Pairs
+#. Let us now add Key-Value Pairs
 
    - Select an Image from the Dataset and in the right-hand pane in ``General`` tab, click the harmonica Key-Value Pairs.
```

Comment:
Earlier in this guide it says "Choose the Tag option in the right-hand General tab", here it calls it "the harmonica", and also below. I think the terminology should be consistent.

Reply:
fixed in https://github.com/ome/omero-guide-introduction/pull/8/commits/b8b2e91d0977eb0b942dd7bd7fc37e5f39bcadd1